README.md

```diff
@@ -57,26 +57,19 @@ In the channel settings, the base currency must be set to EUR because the paymen
    twig:
        ...
        form_themes: [
-           'form/form_gateway_config_row.html.twig'
+           '@PayPlugSyliusPayPlugPlugin/form/form_gateway_config_row.html.twig'
```

Comment:
To avoid copy/paste template. 
Maybe later, we could use a CompilerPass to load it directly